shift_portal/views/res_config_views.xml

```diff
@@ -7,25 +7,128 @@
     <record id="view_website_shift_config_irregular" model="ir.ui.view">
         <field name="name">Website Shift Settings Irregular Worker</field>
         <field name="model">res.config.settings</field>
-        <field name="inherit_id" ref="website.res_config_settings_view_form" />
+        <field name="inherit_id" ref="shift.res_config_settings_shift_view_form" />
+        <!-- priority 21 to put it after website (priority 20) -->
+        <field name="priority" eval="21 " />
```

Comment:
You may want to chose a higher priority so that if something should come between the two, it's possible. Something like 30.

But this change may have undesired effect if there is some other view with other priority, so let's not change it now.

Reply:
OK

Reply:
done